### DISCUSSION TITLE
Extreme amount of information and system windows in Elemenetor

### DISCUSSION THREAD

**Discussion by speedxcz** [elementor/elementor#29351] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Hi, would it be possible to somehow make the system reports that Elementor generates more reasonable. Very often it happens that despite all the reports about updates, new Elementor products etc... nothing is visible but Elementor. It is extremely annoying and from the clients point of view it is extremely unprofessional. We pay for a pro plugin, which then infests the administration with an incredible amount of info windows and similar ballast. Not to mention the extreme amount of spam on Site Mailer, Image Optimizer, AI texts etc..
Thanks a lot in advance.

![elemntor-spam-fog](https://github.com/user-attachments/assets/6d67a55c-5ca9-42f7-9b6c-79cbd44ad9c0)

### Describe the solution you'd like.

Thanks in advance for some more sensible solutions, maybe a pooled information. Or specifying specific users to whom these messages will be displayed, so that they don't bother clients, but only the specific person/people who are troubleshooting the system.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by jazir555** [comment]
I agree this needs to be addressed, but for a band-aid solution right now I'd recommend admin notification center:

https://wordpress.org/plugins/wp-admin-notification-center/
